projects/packages/scheduled-updates/src/class-scheduled-updates.php

```diff
@@ -183,10 +190,12 @@ public static function create_scheduled_update( $timestamp, $interval, $plugins
 	 * @return \WP_Error|bool True on success, WP_Error on failure.
 	 */
 	public static function delete_scheduled_update( $timestamp, $plugins ) {
-		// Be sure to clear the cron cache before removing a cron entry.
+		$ret = wp_unschedule_event( $timestamp, self::PLUGIN_CRON_HOOK, $plugins, true );
+
+		// Be sure to clear the cron cache after removing a cron entry.
 		self::clear_cron_cache();
 
-		return wp_unschedule_event( $timestamp, self::PLUGIN_CRON_HOOK, $plugins, true );
+		return $ret;
```

Comment:
Would this be an opportunity to move `wp_unschedule_event()` back into `delete_item()` [and hook ](https://github.com/Automattic/jetpack/pull/37007/files#diff-16bb61901772bfecbc9020e4b205240d10a347c6f5e6ac61fe795186bcc4d997R62)`clear_cron_cache()` into `jetpack_scheduled_update_deleted`?

Reply:
This idea is excellent, but it can be added in a separate PR.